projects/scripts/adi_project_xilinx.tcl

```diff
@@ -253,6 +253,21 @@ proc adi_project_run {project_name} {
   open_run impl_1
   report_timing_summary -file timing_impl.log
 
+  if {[info exists ::env(ADI_GENERATE_UTILIZATION)]} {
+    if {[catch {xilinx::designutils::report_failfast -csv -file resource_utilization.csv -transpose -no_header -ignore_pr -quiet} issue] != 0 } {
+      puts "GENERATE_REPORTS: tclapp::xilinx::designutils not installed"
+    }
+    set fileId [open "MMCM_and_PLL_usage_and_worst_slacks.csv" w]
```

Comment:
Can we rename the file to something more generic or append the (MMCM/PLL/other) information to the existing "resource_utilization.csv" file?   In the later case you would not need the -transpose option.

Reply:
I think it's a good idea, changing the access argument from w to a+ should do the trick. See [open](https://www.tcl.tk/man/tcl8.0/TclCmd/open.htm)

Reply:
If you're not going to have issues in Jenkins side, parsing the file...

Reply:
I have changed the way files are generated, letting just one file and append to it. I kept the "-transpose" option since, in the end, I need to have two lines, first one with names and second one with values.